ledger_device_sdk/src/lib.rs

```diff
@@ -63,6 +63,9 @@ pub fn exiting_panic(_info: &PanicInfo) -> ! {
 pub use ledger_secure_sdk_sys::buttons;
 pub use ledger_secure_sdk_sys::exit_app;
 
+// re-export include_gif crate as an sdk module
+pub use include_gif;
+
```

Comment:
Yes it is a pretty good idea but it means that `include_gif` is no more an independent package published on crates.io right?
And for all existing apps, after updating to this version of the Rust SDK which includes the `include_gif` as a module the `include_gif` dependency shall be removed? 
Also all `cargo update include_gif` (in GitHub workflow, deployment workflow) shall be removed then.

Reply:
It is still exported as an independent package; this incrementally makes it available to apps.

> And for all existing apps, after updating to this version of the Rust SDK which includes the include_gif as a module the include_gif dependency shall be removed?

Yes, I think it should be removed from the boilerplate (and can be removed from existing apps, after making sure the import is fixed accordingly).
It always makes more sense for apps to use the same version the sdk is using, or you end up with two versions if there's a mismatch.

Good point about the workflows, but I guess it should be only removed once the dependency is removed from existing apps?

Reply:
Don't all apps need to use it at least once for the icon show in `NbglHomeAndSettings`?

Also, the possibility of showing other icons in custom UX flows seems potentially useful.

Reply:
Ok I think I see what you mean, and I agree. I need to make further tests to evaluate the impacts of this modification.
Thank you!